src/cache_inode/cache_inode_init.c

```diff
@@ -50,6 +50,19 @@
  * @return CACHE_INODE_SUCCESS or errors.
  *
  */
+
+extern uint64_t number_of_dirent_entries;
+void*
+dirent_entries_monitor(void *notused)
+{
+  while(1) {
+    if(number_of_dirent_entries > 1000*1000) {
```

Comment:
Increase it much larger value. Do we have an estimate on how much memory will a million entries take.

Reply:
each entry is taking around 100 bytes, I am changing this to 100 million.